### ISSUE TITLE
Not able to install or run application on server

### ISSUE THREAD

**Issue by nileshsonu** [helpdesk-z/helpdeskz-dev#77]
Hi All,
Not able to install or run application on my server.
I have tried 2 ways.
1. Uploaded zip on server, rename helpdesk.php file with database configuration and url, but when I am trying in install using domain/helpdesk/install it still shoes "Whoops! We seem to have hit a snag. Please try again later..."
2. Uploaded installed setup and database from localhost to server with database import and changed configuration and trying to access application, domain/helpdesk_clone but still it is showing "Whoops! We seem to have hit a snag. Please try again later...

Note: I am using AWS instance with ubuntu 24.
I have tried it on other hosting provider like hostinger/godaddy. it is working fine there.

Looking for a solution. please connect on nileshsonu@gmail.com

Thank you.

**Comment by nileshsonu** [comment]
Using php 8.3 
@helpdesk-z @i5o @hpopotter7

**Comment by hpopotter7** [comment]
Hola @nileshsonu,

Con otra version de php lo haz intendado? 8.0, 8.1 ??
La instnacia de ASW es de lightsail o EC2?  Ya viene precargado algun XAMMP  o algo similar ó tu lo instalaste?
Otro punto para revisar el tema de puertos, los abiertos por default de AWS son 80 y 443; pero la Base de datos correria sobre 3306.  

Yo tuve un problema similar con manejar una base datos en la misma instancia, AWS no me permitio convivir la apliación junto con la BD, por lo tanto lo tuve que migrar a un VPS por aparte

**Comment by nileshsonu** [comment]
Yes, I have tried with php 7.4 and 8.1 , 8.2 as well.aws ec2 instance it is.
LAMP installed on the server.
Could you please please guide me to setup helpdeskz on my server?
Let's connect on ***@***.***

**Comment by hpopotter7** [comment]
No puedo ver tu email.

Intenta editar la configuración como viene aca:
[https://stackoverflow.com/questions/9766014/connect-to-mysql-on-amazon-ec2-from-a-remote-server](https://stackoverflow.com/questions/9766014/connect-to-mysql-on-amazon-ec2-from-a-remote-server)

Tambien puedes renombrar los archivos hdz/app/Config/Boot production.php<--->development.php para tener mas detalle del error.

**Comment by gianlucagiacometti** [comment]
I confirm that the app does not work with php8.3

**Comment by Mistik** [comment]
@helpdesk-z -z @i5o  @hpopotter7 where can I contact the current owner of the software/repo?

**Comment by melaxon** [comment]
Check php error log or post it here
